code/game/machinery/machinery.dm

```diff
@@ -116,16 +116,23 @@ Class Procs:
 	var/mob/living/occupant = null
 	var/unsecuring_tool = /obj/item/weapon/wrench
 	var/interact_offline = 0 // Can the machine be interacted with while de-powered.
+	var/fast_process = 0 // Process faster than other machines?

```

Comment:
No!

Override New() to remove itself from machine, and add itself to fast machine